lib/io/src/ui/about_dialog.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:paintroid/io/src/ui/delete_project_dialog.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../../ui/color_schemes.dart';
+
+Future<bool?> showMyAboutDialog(BuildContext context, String version) =>
+    showGeneralDialog<bool>(
+        context: context,
+        pageBuilder: (_, __, ___) => MyAboutDialog(version: version),
+        barrierDismissible: true,
+        barrierLabel: "Show project details dialog box");
+
+class MyAboutDialog extends ConsumerStatefulWidget {
+  final String version;
+
+  const MyAboutDialog({Key? key, required this.version}) : super(key: key);
+
+  @override
+  ConsumerState<MyAboutDialog> createState() => _MyAboutDialogState();
+}
+
+class _MyAboutDialogState extends ConsumerState<MyAboutDialog> {
+  static const license = 'GNU Affero General Public License, v3';
+  static const content =
+      'Pocket Paint is a picture editing library that is part of the Catrobat project.\n\nCatrobat is a visual programming language and set of creativity tools for smartphones.\n\nThe source code of Pocket Paint is mainly licensed under the $license.\nFor precise details of the license see the link below\n';
+  static const urlLicenseDescription = '\nPocket Paint source code license';
+  static const urlCatrobatDescription = '\nAbout Catrobat';
+  static const urlLicense = 'https://developer.catrobat.org/licenses';
+  static const urlCatrobat = 'https://catrobat.org';
+
+  static const urlTextStyle = TextStyle(
+    color: Color(0xFFE68B00),
+    fontSize: 18,
+    decoration: TextDecoration.underline,
+  );
+
+  TextSpan _clickableText(String text, String url, TextStyle? style) =>
```

Comment:
Again function that returns a widget.
Use a class

Reply:
I also thought of using a class for this but actually TextSpan is not a widget 😅

Reply:
True, didn't know that :)